components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/includes/toolbar.html

```diff
@@ -205,7 +205,7 @@
             {% endif %}
 
             {% if not share %}
-            {% if image and not omeTiffDisabled %}
+            {% if image %}
```

Comment:
It seemed so minor as to not justify a separate PR, but I can put it in a separate PR?